Nim/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Welcome to Nim! Stick draw game, 1 to 3 sticks to draw");
+Random random = new Random();
+int matches = 24;
+// string match = "|";''
+
+goback:
+
+Console.WriteLine($"there are this many matches: " + new string('|', matches) + $" ({matches})");
+if (matches > 1)
+{
+    Console.WriteLine("How many matches do you draw?");
+}
+
+
+switch (matches)
+{
+    case > 1:
```

Comment:
This switch is overly compilated, lol. A simple if would've been enough. Also, we like to inverse ifs, to keep our code simple:

```c#
switch(matches)
{
     case > 1:
         // game
     break;
}
// game over
```

Becomes:

```c#
if(matches > 1){
   // game
}
//gameOver
```

And then:

```c#
if(matches <=1){
    // gameOver
    break;
}
// game
```